commands/audit/scarunner.go

```diff
@@ -174,12 +175,26 @@ func getDirectDependenciesFromTree(dependencyTrees []*xrayCmdUtils.GraphNode) []
 	return directDependencies.ToSlice()
 }
 
+func getCurationCacheByTech(tech coreutils.Technology) (string, error) {
+	if tech == coreutils.Maven {
+		return xrayutils.GetCurationMavenCacheFolder(true)
+	}
+	return "", nil
+}
+
 func GetTechDependencyTree(params xrayutils.AuditParams, tech coreutils.Technology) (flatTree *xrayCmdUtils.GraphNode, fullDependencyTrees []*xrayCmdUtils.GraphNode, err error) {
 	logMessage := fmt.Sprintf("Calculating %s dependencies", tech.ToFormal())
+	// in case it's not curation command these params will be empty
+	curationLogMsg, curationCacheFolder, err := getCurationCacheFolderAndLogMsg(params, tech)
+	if err != nil {
+		return
+	}
+	logMessage += curationLogMsg
```

Comment:
```suggestion
	curationLogMsg, curationCacheFolder, err := getCurationCacheFolderAndLogMsg(params, tech)
	if err != nil {
		return
	}
	// In case it's not a curation command these params will be empty
	logMessage += curationLogMsg
```
comments starts with Captial letters.